### DISCUSSION TITLE
Icon versus present weather discrepancy

### DISCUSSION THREAD

**Discussion by sdennstaedt** [weather-gov/api#391] [Q&A]
Appears that in this case for MSY, the present weather icon is showing as rain, when thunderstorms are occurring.  Wouldn't the thunderstorm icon be more appropriate here?

https://api.weather.gov/stations/kmsy/observations/latest?require_qc=true 

Here's some of the json output for this...

    "properties": {
        "@id": "https://api.weather.gov/stations/KMSY/observations/2019-10-21T17:53:00+00:00",
        "@type": "wx:ObservationStation",
        "elevation": {
            "value": 9,
            "unitCode": "unit:m"
        },
        "station": "https://api.weather.gov/stations/KMSY",
        "timestamp": "2019-10-21T17:53:00+00:00",
        "rawMessage": "KMSY 211753Z 22012G20KT 2SM TSRA BR SCT015CB SCT026 BKN037 OVC130 24/22 A2988 RMK AO2 LTG DSNT ALQDS SLP120 OCNL LTGICCG OHD TS OHD MOV NE P0017 60021 T02440222 10261 20244 51012",
        "textDescription": "Thunderstorms and Rain and Fog/Mist",
        "icon": "**https://api.weather.gov/icons/land/day/rain?size=medium**",
        "presentWeather": [
            {
                "intensity": null,
                "modifier": null,
                "weather": "thunderstorms",
                "rawString": "TS"
            },
            {
                "intensity": null,
                "modifier": null,
                "weather": "rain",
                "rawString": "TSRA"
            },
            {
                "intensity": null,
                "modifier": null,
                "weather": "fog_mist",
                "rawString": "BR"
            }
        ],

**Comment by StephenClouse** [comment]
I can confirm this is a bug that we're tracking internally.

**Comment by sdennstaedt** [comment]
Thank you.
